src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -178,7 +178,7 @@ define([
         $.ajax({
             type: "POST",
             dataType: "text",
-            url: "/logout",
```

Comment:
Good catch, didn't realize there's something more to the logout link at the time.